packages/fiori-annotation-api/src/avt/to-internal.ts

```diff
@@ -257,10 +257,10 @@ export function convertApplyToInternal(aliasInfo: AliasInformation, apply: Apply
  */
 export function convertPrimitiveValueToInternal(
     type: string,
-    value: string | number | boolean,
+    value: string | number | boolean | undefined,
     aliasInfo: AliasInformation
 ): string {
-    const text = value.toString();
+    const text = String(value);
```

Comment:
We probably should not return `'undefined'` as a value in case the value does not exist.
```suggestion
    const text = value ? value.toString() : '';
```